### PR TITLE
feat: Display support us

### PR DESCRIPTION
**Pull request by Crash--**
Support us button. Lot of copy / paste. 

Capture d’écran 2019-11-08 à 15 25 37

### CODE REVIEW

File: src/components/pushClient/Supportus.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import { withClient } from 'cozy-client'
+import withInstance from 'drive/web/modules/upload/withInstance'
+import withDiskUsage from './withDiskUsage'
+
+import { buildPremiumLink } from 'drive/web/modules/upload/QuotaAlert'
+import { isMobileApp } from 'cozy-device-helper'
+import get from 'lodash/get'
+
+const Supportus = ({ t, client }) => {
```
Comment (y-lohse) [inline src/components/pushClient/Supportus.jsx]: Nit: SupportUs, inthe filename too?

File: src/components/pushClient/Supportus.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    instanceInfo,
+    'context.data.attributes.enable_premium_links'
+  )
+  const diskUsage = withDiskUsage(client)
+
+  if (
+    enable_premium_links &&
+    managerUrl &&
+    uuid &&
+    diskUsage.data.attributes.quota < 50000000
```
Comment (y-lohse) [inline src/components/pushClient/Supportus.jsx]: How much is that ? 😄

File: src/components/pushClient/withDiskUsage.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const withDiskUsage = client => {
+  const [diskusage, setDiskusage] = useState()
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const diskusageResult = await client
+          .getStackClient()
+          .fetchJSON('GET', '/settings/disk-usage')
```
Comment (y-lohse) [inline src/components/pushClient/withDiskUsage.jsx]: Note that there is a SettingsCollection now in cozy-client!

File: src/components/pushClient/Supportus.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+const Supportus = ({ t, client }) => {
+  if (isMobileApp()) return null
+
+  const instanceInfo = withInstance(client)
+
+  const uuid = get(instanceInfo, 'instance.data.attributes.uuid')
+  const managerUrl = get(instanceInfo, 'context.data.attributes.manager_url')
+  const enable_premium_links = get(
+    instanceInfo,
+    'context.data.attributes.enable_premium_links'
```
Comment (y-lohse) [inline src/components/pushClient/Supportus.jsx]: Is this set to false when the user has a premium subscription?

Comment (Crash--) [reply src/components/pushClient/Supportus.jsx]: added comment in https://github.com/cozy/cozy-drive/pull/1891/commits/b442343f4a21804f787cf5d64b84af6734b44a28

**Pull request merged**